src/admin-views/settings/tickets-commerce/gateways/toggle.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Template to display a featured gateway.
+ *
+ * @since TBD
+ *
+ * @var Tribe__Tickets__Admin__Views $this              Template object.
```

Comment:
Here it's not an instance of `Tribe__Tickets__Admin__Views` it's an instance of `Tribe__Template`